### PR TITLE
`APITester`: added `StoreProduct` API

### PR DESCRIPTION
**Pull request by NachoSoto**

### CODE REVIEW

File: APITesters/ObjCAPITester/ObjCAPITester/main.m [hunk 1]
```diff
@@ -18,6 +18,7 @@
 #import "RCPackageAPI.h"
 #import "RCRefundRequestStatusAPI.h"
 #import "RCTransactionAPI.h"
+#import "RCStoreProductAPI.h"
```
Comment (taquitos) [inline APITesters/ObjCAPITester/ObjCAPITester/main.m:21]: Super tiny nit, but alphabetic order would be :chefskiss:

**Review by taquitos — APPROVED** [review]
🌮  👍

File: APITesters/ObjCAPITester/ObjCAPITester/main.m (hunk 1 above)
Comment (NachoSoto) [reply APITesters/ObjCAPITester/ObjCAPITester/main.m:21]: Oh sorry I missed this. I'll do it in a follow up

Comment (NachoSoto) [reply APITesters/ObjCAPITester/ObjCAPITester/main.m:21]: Done in #1138.

**Pull request merged**